### PR TITLE
Add header and basic styling

### PR DESCRIPTION
**Pull request by jessdejong**
Added a header to my portfolio website, and some basic styling for my profile photo. Added my profile photo into the header.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,11 +2,23 @@
...
+    <div class="header">
+      <img src="images/Profile_photo_Jess.jpg" alt="Profile photo of Jess" height="250">
+      <h1>Jess's Portfolio</h1>
+      <strong>
+        <a href="#">About Me</a>
+        <a href="#">Projects</a>
+        <a href="#">Experience</a>
+        <a href="#">Hobbies</a>
+        <a href="#">Socials & Contact</a>
+      </strong>
```
Comment (daniel-google) [inline portfolio/src/main/webapp/index.html:19]: nit: consider using a styled list for this, e.g.

```html
<ul class="navigation">
  <li><a href="#">About Me</a>
  <!-- ... -->
</ul>
```

```css
.navigation li {
  list-style: none;
  display: inline-block;
}
```

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -7,3 +7,54 @@
...
+}
+
+/* Stylize image in header as a profile photo */
+
+.header img {
+  float: left;
+  border-radius: 70%;
+  border: 2px solid black;
+  height: 150px;
+  width: 150px;
```
Comment (daniel-google) [inline portfolio/src/main/webapp/style.css:34]: nit: if you omit one of these, it'll preserve the aspect ratio of the image (assuming it's not square)

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -7,3 +7,54 @@
...
+/* Interactive header links */
+
+.header a {
+  margin-left: 5px;
+  float: left;
+  color: black;
+  text-align: center;
+  padding: 12px;
+  text-decoration: none;
+  font-size: 18px;
```
Comment (daniel-google) [inline portfolio/src/main/webapp/style.css:47]: nit: font sizes should normally be specified in "em" units, which will scale with the user's font size setting in their browser

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (jessdejong) [reply portfolio/src/main/webapp/index.html:19]: This looks nicer, I'll look into it!

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (jessdejong) [reply portfolio/src/main/webapp/style.css:47]: That makes sense, I'll make the change.

**Pull request merged**